tsconfig.build.json

```diff
@@ -1,7 +1,6 @@
 {
   "compilerOptions": {
     "declaration": true,
-    "declarationMap": true,
```

Comment:
Does this need to be removed? It's nice to have this for debugging.

Reply:
Must have been deleted my something automatically? will make sure it's re added